src/plugins/fakeProfileThemesAndEffects/index.tsx

```diff
@@ -0,0 +1,496 @@
+    const themeColors: [number, number] = [-1, -1];
+    let numSignIndex: number = -1;
+    for (let i: number = 0; i < bio.length; i++) {
+        if (bio[i] === "#") {
+            numSignIndex = i;
+            break;
+        }
+    }
+    if (numSignIndex !== -1) {
+        let tempStr: string = "";
+        let lim: number = numSignIndex + 7 < bio.length ? numSignIndex + 7 : bio.length;
+        for (let i: number = numSignIndex + 1; i < lim; i++) {
+            if (bio[i] === "," || bio[i] === "]") break;
+            tempStr += bio[i];
+        }
+        let color: number = parseInt(tempStr, 16);
+        if (!Number.isNaN(color)) {
+            themeColors[0] = color;
+            numSignIndex = -1;
+            for (let i: number = lim; i < bio.length; i++) {
+                if (bio[i] === "#") {
+                    numSignIndex = i;
+                    break;
+                }
+            }
+            if (numSignIndex !== -1) {
+                tempStr = "";
+                lim = numSignIndex + 7 < bio.length ? numSignIndex + 7 : bio.length;
+                for (let i: number = numSignIndex + 1; i < lim; i++) {
+                    if (bio[i] === "]" || bio[i] === ",") break;
+                    tempStr += bio[i];
+                }
+                color = parseInt(tempStr, 16);
+                if (!Number.isNaN(color))
+                    themeColors[1] = color;
+            }
+        }
+    }
+    return themeColors;
+}
+
+function legacy3y3(user: UserProfile, str: string): void {
+    const themeColors: [number, number] = legacyStrToThemeColors(str);
+    if (themeColors[0] > -1) {
+        if (themeColors[1] > -1)
+            user.themeColors = [themeColors[0], themeColors[1]];
+        else
+            user.themeColors = [themeColors[0], themeColors[0]];
+        user.premiumType = 2;
+    } else if (themeColors[1] > -1) {
+        user.themeColors = [themeColors[1], themeColors[1]];
+        user.premiumType = 2;
+    }
+}
+
+function updateUserThemeColors(user: UserProfile, primary: number, accent: number): void {
+    if (primary > -1) {
+        if (accent > -1)
+            user.themeColors = [primary, accent];
+        else
+            user.themeColors = [primary, primary];
+        user.premiumType = 2;
+    } else if (accent > -1) {
+        user.themeColors = [accent, accent];
+        user.premiumType = 2;
+    }
+}
+
+function updateUserEffectID(user: UserProfile, id: bigint): void {
+    if (id > -1n) {
+        user.profileEffectID = id.toString();
+        user.premiumType = 2;
+    }
+}
+
+const settings = definePluginSettings({
+    prioritizeNitro: {
+        description: "Source to use if profile theme colors / effects are set by both Nitro and About Me",
+        type: OptionType.SELECT,
+        options: [
+            { label: "Nitro", value: true },
+            { label: "About Me", value: false, default: true },
+        ]
+    },
+    "Hide 3y3 Builder": {

```

Comment:
```suggestion
    hide3y3Builder: {
```
and change things accessing this

Reply:
This must be done otherwise the setting name will show as "Hide3y3 Builder" in the plugin settings modal.